metasploit-framework.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   # only do a git ls-files if the .git folder exists and we have a git binary in PATH
   if File.directory?(File.join(File.dirname(__FILE__), ".git")) && Msf::Util::Helper.which("git")
     spec.files         = `git ls-files`.split($/).reject { |file|
-      file =~ /^documentation|^external/
+      file =~ /^external/
```

Comment:
Not a blocker, but I think I'm in favor of moving the swagger ruby files into the lib folder / replacing `load`s with `require`s etc

Reply:
Thanks for the feedback on that, checkout the updated revisions to see if you agree on the approach and namespacing I used to do this.